lib/Test.js

```diff
@@ -256,6 +257,10 @@ define([
 				})
 				.then(function () {
 					self.hasPassed = true;
+					if (self._usesRemote && !self.isAsync) {
+						console.warn("This test uses the remote but isn't marked as async! Intern won't wait for your test actions to complete.");
```

Comment:
So should this just be a this.report('warnAsync') or something? I'm not sure how that part of the code works.